packages/angular/.eslintrc.cjs

```diff
@@ -3,5 +3,8 @@ module.exports = {
     browser: true,
   },
   extends: ['../../.eslintrc.js'],
+  rules: {
+    '@sentry-internal/sdk/no-optional-chaining': 'off',
```

Comment:
we do not want to turn this off - see https://blog.sentry.io/performance-impact-of-generated-javascript/#removing-optional-chaining